package/kernel/linux/modules/usb.mk

```diff
@@ -1340,6 +1340,7 @@ $(eval $(call KernelPackage,usb-net-rtl8150))
 
 define KernelPackage/usb-net-rtl8152
   TITLE:=Kernel module for USB-to-Ethernet Realtek convertors
+  DEPENDS:=+LINUX_5_10:kmod-crypto-sha256 +LINUX_5_10:r8152-firmware
```

Comment:
Please use +!LINUX_5_4:... instead

Reply:
`request_firmware` was added in v5.5 unless is backported or merged in 5.4 will not work.

Reply:
How that relevant? I just wanted to invert logic so we don't have to touch this again when the next kernel comes. Or will this be removed again after 5.10?

Reply:
I've done the suggested changes, since this will not be removed after kernel 5.10